carma_v2x_msgs/msg/EmergencyVehicleAck.msg

```diff
@@ -0,0 +1,10 @@
+# EmergencyVehicleAck.msg
+
+# This message is used by an emergency vehicle to acknowledge receipt of an 
+# EmergencyVehicleResponse message from a carma-equipped vehicle
+
+# standard header for all mobility messages
+carma_v2x_msgs/MobilityHeader m_header
+
+# True when the emergency vehicle has received the EVR message. Currently this should never be false
```

Comment:
Let's spell out "EmergencyVehicleResponse" message here just in case the user misses the full message name in the message description included earlier in the file.